docs/resources/aviatrix_azure_spoke_native_peering.md

```diff
@@ -30,7 +30,7 @@ The following arguments are supported:
 * `transit_gateway_name` - (Required) Name of an Transit FireNet-enabled Azure transit gateway.
 * `spoke_account_name` - (Required) An Aviatrix account that corresponds to a subscription in Azure.
 * `spoke_region` - (Required) Spoke VNet region. Example: "West US".
-* `spoke_vpc_id` - (Required) Combination of the Spoke's VNet name and resource group. Example: "Foo_VNet:Bar_RG".
+* `spoke_vpc_id` - (Required) Combination of the Spoke's VNet name, resource group and GUID. Example: "Foo_VNet:Bar_RG:GUID". Available as of provider version R2.20.1.
```

Comment:
I'm not sure we need the `Available as of provider version R2.20.1.` because this attribute was already available before 2.20.1. Instead we could say something like "As of controller version 6.5 spoke_vpc_id must include the GUID of the vpc"